spring-integration-amqp/src/test/java/org/springframework/integration/amqp/rule/BrokerRunning.java

```diff
@@ -132,12 +132,15 @@ public Statement apply(Statement base, Description description) {
 		return super.apply(base, description);
 	}
 
-	public void removeTestQueues() {
+	public void removeTestQueues(String... additionalQueues) {
```

Comment:
Moved AMQP test fix to https://github.com/spring-projects/spring-integration/pull/2064